ts/private/ts_project.bzl

```diff
@@ -84,6 +93,9 @@ See https://github.com/aspect-build/rules_ts/issues/228 for more details.
         execution_requirements["worker-key-mnemonic"] = "TsProject"
         executable = ctx.executable.tsc_worker
 
+    # Add all arguments from options first to allow users override them via args.
+    arguments.add_all(options.args)
```

Comment:
note, this means you won't be able to see the `args=["--skipLibCheck"]` when you `bazel query //the:target` now, like you could previously. But since we already "magically add" to the args here I think it's fine to change from macro to rule doing this.